README.md

```diff
@@ -75,14 +87,34 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('/', function (req, res){
+  res.send('')
+})
+app.post('/', function (req, res) {
+  res.send('')
+})
+app.put('/', function(res,req) {
+    res.send('');
+}, send);
+
+app.delete(function(req, res) {
+        object.remove({
+            _id: req.params.object_id
+        }, function(err, ) {
+            if (err)
+                res.send(err);
+
+            res.json({ message: 'Successfully deleted' });
+        });
+    });
 
 ```
 
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
-```text
+```Express allows you to write server-side applications in JS, while rails. Rails has to be built out in the files and implemented to what specific file its to be used in. the framework in Rails is opinionated in that if you work on someone elses prior work itll be easier to work with language wise.
```

Comment:
Hm, not sure I see the differences here. Yes, Rails is opinionated -- so what is Express? What makes it so?